### ISSUE TITLE
[FEA] Support FIRST_VALUE and LAST_VALUE in grouped_rolling_window

### ISSUE THREAD

**Issue by williamBlazing** [rapidsai/cudf#7231]
**Is your feature request related to a problem? Please describe.**
Like ROW_NUMBER, which supported in PR 4881, FIRST_VALUE is widely used window function in sql. So, we want to provide it for BlazingSQL. FIRST_VALUE and LAST_VALUE are fairly trivial where there are no partitions to worry about, but if you have multiple groups or partitions, for the case of `grouped_rolling_window`, then its not so trivial, and we would want native support for that aggregator in cudf.

**Describe the solution you'd like**
Implementation of the aggregation for FIRST_VALUE and LAST_VALUE, especially for `grouped_rolling_window`

**Comment by revans2** [comment]
Would it make since to use the existing make_nth_element_aggregation for this?  This is how we implement first and last for hash aggregates.

**Comment by Christian8491** [comment]
@revans2, Is it possible to get some unit tests from `rolling_test.cpp` using `make_nth_element_aggregation` ?. When I tried to use this aggregation with `grouped_rolling_window().` I got  this error: `../rolling_detail.cuh:738: Aggregation operator and/or input type combination is invalid`

**Comment by revans2** [comment]
@Christian8491 I don't think that `make_nth_element_aggregation` is supported by grouped rolling window yet.  I was mostly suggesting that perhaps we should reuse the aggregation for this use case instead of creating a new one.
